src/components/molecules/Dashboard/index.tsx

```diff
@@ -1,19 +1,45 @@
 import React from "react";
 
+import Loading from "@reearth/components/atoms/Loading";
 import { styled } from "@reearth/theme";
 
 export * from "./types";
 
 export type Props = {
   className?: string;
   header?: React.ReactNode;
+  isLoading?: boolean;
+  hasMoreProjects?: boolean;
+  onGetMoreProjects?: () => void;
 };
+function handleScroll(
+  { currentTarget }: React.UIEvent<HTMLDivElement, UIEvent>,
+  onLoadMore?: () => void,
+) {
+  if (currentTarget.scrollTop + currentTarget.clientHeight >= currentTarget.scrollHeight) {
+    onLoadMore?.();
+  }
+}
```

Comment:
Can you move handleScroll to `utils` and import it in where needed? We don't want to be copy/pasting it all over.